src/login/logind-dbus.c

```diff
@@ -3080,9 +3080,17 @@ int manager_start_scope(
         if (r < 0)
                 return r;
 
-        r = sd_bus_message_append(m, "(sv)", "TasksMax", "t", tasks_max);
-        if (r < 0)
-                return r;
+        if (more_properties) {
+                r = sd_bus_message_copy(m, more_properties, true);
+                /* XXX: r == 1 here, OK. */
+                if (r < 0)
+                        return r;
+        } else {
+                // TODO /* disable TasksMax= for the session scope, rely on the slice setting for it */
+                r = sd_bus_message_append(m, "(sv)", "TasksMax", "t", (uint64_t)-1);
+                if (r < 0)
+                        return bus_log_create_error(r);
```

Comment:
please set TasksMax unconditionally, before appending `more_properties`. People can override it with that, since props later in the array override the same props earlier in the array just fine.